### ISSUE TITLE
System.Numerics.Tensors.Tests fails with assertion in mini-ppc.c on ppc64le

### ISSUE THREAD

**Issue by akoeplinger** [dotnet/runtime#97296]
Disabled the test project in the meantime, it started after https://github.com/dotnet/runtime/pull/97192 was merged. Note that there's another issue with the Mono JIT so I had to disable the tests on Mono in general: https://github.com/dotnet/runtime/issues/97295

Probably similar to https://github.com/dotnet/runtime/issues/71080

```
  Discovering: System.Numerics.Tensors.Tests (method display = ClassAndMethod, method display options = None)
  Discovered:  System.Numerics.Tensors.Tests (found 4087 test cases)
  Starting:    System.Numerics.Tensors.Tests (parallel test collections = on, max threads = 2)
* Assertion at /__w/1/s/src/mono/mono/mini/mini-ppc.c:5205, condition `ppc_is_imm16 (inst->inst_offset)' not met


=================================================================
	Native Crash Reporting
=================================================================
Got a SIGABRT while executing native code. This usually indicates
a fatal error in the mono runtime or one of the native libraries 
used by your application.
=================================================================

=================================================================
	Native stacktrace:
=================================================================
	0x78bd0c28f868 - Unknown
	0x78bd0c23e39c - Unknown
	0x78bd0c28ed28 - Unknown
	0x78bd0cd604c8 - Unknown
	0x78bd0c6cd168 - Unknown
	0x78bd0c6a4850 - Unknown
	0x78bd0c3682d8 - Unknown
	0x78bd0c37e3f4 - Unknown
	0x78bd0c379974 - Unknown
	0x78bd0c36888c - Unknown
	0x78bd0c3689dc - Unknown
	0x78bd0c368a5c - Unknown
	0x78bd0c288064 - Unknown
	0x78bd0c173a8c - Unknown
	0x78bd0c1774d8 - Unknown
	0x78bd0c178f58 - Unknown
	0x78bd0c17fedc - Unknown
	0x78bd0c17eca8 - Unknown
	0x78bd0c241714 - Unknown
	0x78bd0c2411c8 - Unknown
	0x78bd0bcd01c4 - Unknown
	0x78bcdfdaea78 - Unknown
	0x78bcdfdae990 - Unknown
	0x78bcdfdac0a4 - Unknown
	0x78bcdfdabdbc - Unknown
	0x78bd008c32ec - Unknown
	0x78bd008c3100 - Unknown
	0x78bd008c2f58 - Unknown
	0x78bcdfdab9cc - Unknown
	0x78bd0b878570 - Unknown
	0x78bd087df9f4 - Unknown
	0x78bd0880fd6c - Unknown
	0x78bd087d9320 - Unknown
	0x78bd087d8dc8 - Unknown
	0x78bd00892224 - Unknown
	0x78bd00890e38 - Unknown
	0x78bd0088f3d8 - Unknown
	0x78bd0088f244 - Unknown
	0x78bd0088f150 - Unknown
	0x78bd0088daf0 - Unknown
	0x78bd0088d808 - Unknown
	0x78bd0088d624 - Unknown
	0x78bd0088d29c - Unknown
	0x78bd0088c6c4 - Unknown
	0x78bd0088bae0 - Unknown
	0x78bd0088a7b0 - Unknown
	0x78bd0088897c - Unknown
	0x78bd00887888 - Unknown
	0x78bd008876ec - Unknown
	0x78bd008875f4 - Unknown
	0x78bd00886a38 - Unknown
	0x78bd0088026c - Unknown
	0x78bd0087d858 - Unknown
	0x78bd0087cea4 - Unknown
	0x78bd0087cdb0 - Unknown
	0x78bd0087c66c - Unknown
	0x78bd0087b858 - Unknown
	0x78bd0087b1d0 - Unknown
	0x78bd0087ad7c - Unknown
	0x78bd008793bc - Unknown
	0x78bd008783ec - Unknown
	0x78bd008777f0 - Unknown
	0x78bd00877470 - Unknown
	0x78bd00876e58 - Unknown
	0x78bd0087642c - Unknown
	0x78bd00874b08 - Unknown
	0x78bd00874974 - Unknown
	0x78bd0087480c - Unknown
	0x78bd00873334 - Unknown
	0x78bd0086f420 - Unknown
	0x78bd0086f27c - Unknown
	0x78bd0086f178 - Unknown
	0x78bd008c1618 - Unknown
	0x78bd008c1178 - Unknown
	0x78bd008c0f64 - Unknown
	0x78bd0086b5f8 - Unknown
	0x78bd0086af68 - Unknown
	0x78bd0086adcc - Unknown
	0x78bd0086acc4 - Unknown
	0x78bd00869c1c - Unknown
	0x78bd008697bc - Unknown
	0x78bd00868f3c - Unknown
	0x78bd00868b50 - Unknown
	0x78bd008689ac - Unknown
	0x78bd008688a8 - Unknown
	0x78bd008677b4 - Unknown
	0x78bd00867350 - Unknown
	0x78bd008671bc - Unknown
	0x78bd008670b8 - Unknown
	0x78bd00866b4c - Unknown
	0x78bd0085d9c4 - Unknown
	0x78bd0085cc48 - Unknown
	0x78bd0085ca9c - Unknown
	0x78bd0085c994 - Unknown
	0x78bd00857ae8 - Unknown
	0x78bd00857418 - Unknown
	0x78bd0085727c - Unknown
	0x78bd00857174 - Unknown
	0x78bd00856a70 - Unknown
	0x78bd0085591c - Unknown
	0x78bd00855138 - Unknown
	0x78bd00854f94 - Unknown
	0x78bd00854e90 - Unknown
	0x78bd008505d8 - Unknown
	0x78bd0084d478 - Unknown
	0x78bd0084cec4 - Unknown
	0x78bd0084cdbc - Unknown
	0x78bd0084b9e4 - Unknown
	0x78bd0084b5c4 - Unknown
	0x78bd0084b0e4 - Unknown
	0x78bd0194c2d0 - Unknown
	0x78bd0194be14 - Unknown
	0x78bd0194b03c - Unknown
	0x78bd0084a8c0 - Unknown
	0x78bd0084a538 - Unknown
	0x78bd0084a424 - Unknown
	0x78bd00849f18 - Unknown
	0x78bd0194be14 - Unknown
	0x78bd0083bbfc - Unknown
	0x78bd0b870c3c - Unknown
	0x78bd00849e4c - Unknown
	0x78bd0083db3c - Unknown
	0x78bd0194c884 - Unknown
	0x78bd0194c718 - Unknown
	0x78bd0194c2d0 - Unknown
	0x78bd0194be14 - Unknown
	0x78bd0194b03c - Unknown
	0x78bd01949ab8 - Unknown
	0x78bd019498dc - Unknown
	0x78bd019495ec - Unknown
	0x78bd0b963e34 - Unknown
	0x78bd0c184c88 - Unknown
	0x78bd0c43ba48 - Unknown
	0x78bd0c4640a0 - Unknown
	0x78bd0cd08838 - Unknown
	0x78bd0c7cba44 - Unknown

=================================================================
	External Debugger Dump:
=================================================================

=================================================================
	Basic Fault Address Reporting
=================================================================
Memory around native instruction pointer (0x78bd0c6cd168):0x78bd0c6cd158  02 00 60 38 00 00 a0 38 08 00 c0 38 02 00 00 44  ..`8...8...8...D
0x78bd0c6cd168  26 00 00 7c 28 01 21 e9 f0 8f 4d e9 79 52 29 7d  &..|(.!...M.yR)}
0x78bd0c6cd178  00 00 40 39 3c 00 82 40 50 01 21 38 78 fb e3 7f  ..@9<..@P.!8x...
0x78bd0c6cd188  10 00 01 e8 e8 ff a1 eb f0 ff c1 eb f8 ff e1 eb  ................

=================================================================
	Managed Stacktrace:
=================================================================
	  at <unknown> <0xffffffff>
	  at <unknown> <0xffffffff>
	  at System.Numerics.Tensors.TensorPrimitives:InvokeSpanScalarIntoSpan <0x00090>
	  at System.Numerics.Tensors.TensorPrimitives:Max <0x0008c>
	  at SpanScalarDestinationDelegate`3:invoke_void_ReadOnlySpan`1<T1>_T2_Span`1<T3> <0x00118>
	  at <>c__DisplayClass40_0:<SpanScalarDestination_ThrowsForOverlapppingInputsWithOutputs>b__0 <0x00218>
	  at Xunit.Assert:RecordException <0x000b8>
	  at Xunit.Assert:Throws <0x0008c>
	  at System.AssertExtensions:Throws <0x0008c>
	  at System.Numerics.Tensors.Tests.GenericFloatingPointNumberTensorPrimitivesTests`1:SpanScalarDestination_ThrowsForOverlapppingInputsWithOutputs <0x00260>
	  at System.Object:InvokeStub_GenericFloatingPointNumberTensorPrimitivesTests`1.SpanScalarDestination_ThrowsForOverlapppingInputsWithOutputs <0x000bc>
	  at <Module>:invoke_object_object_Span`1<object> <0x00108>
	  at System.Reflection.MethodBaseInvoker:InvokeWithFewArgs <0x00570>
	  at System.Reflection.RuntimeMethodInfo:Invoke <0x00534>
	  at System.Reflection.MethodBase:Invoke <0x0007c>
	  at Xunit.Sdk.TestInvoker`1:CallTestMethod <0x00088>
	  at <<InvokeTestMethodAsync>b__1>d:MoveNext <0x00694>
	  at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00134>
	  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder:Start <0x00070>
	  at <>c__DisplayClass48_0:<InvokeTestMethodAsync>b__1 <0x0011c>
	  at <AggregateAsync>d__4:MoveNext <0x00124>
	  at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x0012c>
	  at Xunit.Sdk.ExecutionTimer:AggregateAsync <0x00130>
	  at <>c__DisplayClass48_0:<InvokeTestMethodAsync>b__0 <0x001d0>
	  at <RunAsync>d__9:MoveNext <0x000b0>
	  at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00124>
	  at Xunit.Sdk.ExceptionAggregator:RunAsync <0x0012c>
	  at <InvokeTestMethodAsync>d__48:MoveNext <0x00390>
	  at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x0013c>
	  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:Start <0x00070>
	  at Xunit.Sdk.TestInvoker`1:InvokeTestMethodAsync <0x00160>
	  at Xunit.Sdk.XunitTestInvoker:InvokeTestMethodAsync <0x00214>
	  at <<RunAsync>b__47_0>d:MoveNext <0x00888>
	  at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00144>
	  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:Start <0x00070>
	  at Xunit.Sdk.TestInvoker`1:<RunAsync>b__47_0 <0x00124>
	  at <RunAsync>d__10`1:MoveNext <0x000b8>
	  at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00124>
	  at Xunit.Sdk.ExceptionAggregator:RunAsync <0x0012c>
	  at Xunit.Sdk.TestInvoker`1:RunAsync <0x00158>
	  at Xunit.Sdk.XunitTestRunner:InvokeTestMethodAsync <0x00138>
	  at <InvokeTestAsync>d__4:MoveNext <0x00258>
	  at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00134>
	  at Xunit.Sdk.XunitTestRunner:InvokeTestAsync <0x00134>
	  at <>c__DisplayClass43_0:<RunAsync>b__0 <0x0006c>
	  at <RunAsync>d__10`1:MoveNext <0x000d0>
	  at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00134>
	  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:Start <0x00070>
	  at Xunit.Sdk.ExceptionAggregator:RunAsync <0x00148>
	  at <RunAsync>d__43:MoveNext <0x00698>
	  at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00144>
	  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:Start <0x00070>
	  at Xunit.Sdk.TestRunner`1:RunAsync <0x00124>
	  at <RunTestAsync>d__11:MoveNext <0x002ac>
	  at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00144>
	  at Xunit.Sdk.XunitTheoryTestCaseRunner:RunTestAsync <0x00100>
	  at <RunAsync>d__19:MoveNext <0x004cc>
	  at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x0013c>
	  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:Start <0x00070>
	  at Xunit.Sdk.TestCaseRunner`1:RunAsync <0x00120>
	  at Xunit.Sdk.XunitTheoryTestCase:RunAsync <0x00150>
	  at Xunit.Sdk.XunitTestMethodRunner:RunTestCaseAsync <0x00108>
	  at <RunTestCasesAsync>d__32:MoveNext <0x00228>
	  at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00144>
	  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:Start <0x00070>
	  at Xunit.Sdk.TestMethodRunner`1:RunTestCasesAsync <0x00124>
	  at <RunAsync>d__31:MoveNext <0x002a0>
	  at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00134>
	  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:Start <0x00070>
	  at Xunit.Sdk.TestMethodRunner`1:RunAsync <0x0011c>
	  at Xunit.Sdk.XunitTestClassRunner:RunTestMethodAsync <0x00188>
	  at <RunTestMethodsAsync>d__38:MoveNext <0x00a78>
	  at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x0014c>
	  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:Start <0x00070>
	  at Xunit.Sdk.TestClassRunner`1:RunTestMethodsAsync <0x00128>
	  at <RunAsync>d__37:MoveNext <0x0050c>
	  at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x0013c>
	  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:Start <0x00070>
	  at Xunit.Sdk.TestClassRunner`1:RunAsync <0x00120>
	  at Xunit.Sdk.XunitTestCollectionRunner:RunTestClassAsync <0x0018c>
	  at <RunTestClassesAsync>d__28:MoveNext <0x004e0>
	  at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00144>
	  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:Start <0x00070>
	  at Xunit.Sdk.TestCollectionRunner`1:RunTestClassesAsync <0x00124>
	  at <RunAsync>d__27:MoveNext <0x0050c>
	  at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x0013c>
	  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:Start <0x00070>
	  at Xunit.Sdk.TestCollectionRunner`1:RunAsync <0x00120>
	  at Xunit.Sdk.XunitTestAssemblyRunner:RunTestCollectionAsync <0x00168>
	  at <>c__DisplayClass14_2:<RunTestCollectionsAsync>b__2 <0x00098>
	  at System.Threading.Tasks.Task`1:InnerInvoke <0x000d8>
	  at <>c:<.cctor>b__281_0 <0x00064>
	  at System.Threading.ExecutionContext:RunInternal <0x00158>
	  at System.Threading.Tasks.Task:ExecuteWithThreadLocal <0x00450>
	  at System.Threading.Tasks.Task:ExecuteEntry <0x00194>
	  at <>c:<.cctor>b__8_0 <0x000ac>
	  at Xunit.Sdk.MaxConcurrencySyncContext:RunOnSyncContext <0x000b0>
	  at <>c__DisplayClass11_0:<WorkerThreadProc>b__0 <0x00094>
	  at System.Threading.ExecutionContext:RunInternal <0x00158>
	  at System.Threading.ExecutionContext:Run <0x000a8>
	  at System.Object:lambda_method2 <0x000d8>
	  at Xunit.Sdk.ExecutionContextHelper:Run <0x000c8>
	  at Xunit.Sdk.MaxConcurrencySyncContext:WorkerThreadProc <0x00378>
	  at <>c:<QueueUserWorkItem>b__5_0 <0x000b0>
	  at System.Threading.Tasks.Task:InnerInvoke <0x0011c>
	  at <>c:<.cctor>b__281_0 <0x00064>
	  at System.Threading.ExecutionContext:RunInternal <0x00158>
	  at System.Threading.Tasks.Task:ExecuteWithThreadLocal <0x00450>
	  at System.Threading.Tasks.Task:ExecuteEntryUnsafe <0x00144>
	  at <>c:<.cctor>b__10_0 <0x000b0>
	  at System.Threading.Thread:StartCallback <0x001c0>
	  at System.Object:runtime_invoke_void__this__ <0x00110>
=================================================================
./RunTests.sh: line 179: 90305 Aborted                 (core dumped) "$RUNTIME_PATH/dotnet" exec --runtimeconfig System.Numerics.Tensors.Tests.runtimeconfig.json --depsfile System.Numerics.Tensors.Tests.deps.json xunit.console.dll System.Numerics.Tensors.Tests.dll -xml testResults.xml -nologo -nocolor -notrait category=IgnoreForCI -notrait category=OuterLoop -notrait category=failing $RSP_FILE
```

/cc @nealef @uweigand

**Comment by matouskozak** [comment]
`System.Runtime.Intrinsics.Tests` failing with the same error and platform e.g. https://dev.azure.com/dnceng-public/public/_build/results?buildId=740764

**Comment by akoeplinger** [comment]
@alhad-deshpande would you mind taking a look?

**Comment by alhad-deshpande** [comment]
@akoeplinger 
Yes, sure will take a look.

**Comment by alhad-deshpande** [comment]
@akoeplinger 
Fixed this via #107678.
